src/test/scala/it/bitrock/dvs/api/routes/RoutesSpec.scala

```diff
@@ -1,5 +1,6 @@
 package it.bitrock.dvs.api.routes
 
+import RoutesSpec._
```

Comment:
Why not full path import?

Reply:
it's the companion object. no problem, let me add the path.